lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -500,14 +539,16 @@ DINode *SPIRVToLLVMDbgTran::transTypeMember(const SPIRVExtInst *DebugInst) {
   assert(Ops.size() >= MinOperandCount && "Invalid number of operands");
 
   DIFile *File = getFile(Ops[SourceIdx]);
-  unsigned LineNo = Ops[LineIdx];
+  SPIRVWord LineNo = getConstantOrLiteral(Ops, LineIdx,
+                                          DebugInst->getExtSetKind());
   StringRef Name = getString(Ops[NameIdx]);
   DIScope *Scope = getScope(BM->getEntry(Ops[ParentIdx]));
   DIType *BaseType =
       transDebugInst<DIType>(BM->get<SPIRVExtInst>(Ops[TypeIdx]));
   uint64_t OffsetInBits =
       BM->get<SPIRVConstant>(Ops[OffsetIdx])->getZExtIntValue();
-  unsigned SPIRVFlags = Ops[FlagsIdx];
+  SPIRVWord SPIRVFlags = getConstantOrLiteral(Ops, FlagsIdx,
```

Comment:
Maybe to leave unsigned instead of SPIRVWord?

Reply:
We should put either `unsigned` or `SPIRVWord` for every `getConstantOrLiteral` call. During implementation I changed type for almost every Debug instruction parameter to `unsigned`, but then convinced myself the SPIR-V specific alias for unsigned `SPIRVWord` is a bit preferable, since we are reading SPIR-V module and expect the parameter value not to overflow a size of a word, leaving some header to define this size.